setup.py

```diff
@@ -22,7 +22,7 @@ def get_version(fname="flake8_bandit.py"):
 VERSION = get_version()
 
 # What packages are required for this module to be executed?
-REQUIRED = ["flake8", "bandit>=1.7.3", "flake8-polyfill", "pycodestyle"]
+REQUIRED = ["flake8>=5.0.0", "bandit>=1.7.3", "flake8-polyfill", "pycodestyle"]
```

Comment:
FWIW neither flake8-polyfill nor pycodestyle are used anywhere in this repo; maybe we could remove the dependency at the same time?

Reply:
Done!